api/app/Entity/Event.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Entity;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Event extends Model
+{
+    protected $fillable = [
+        'event_type_id',
+        'invitee_name',
+        'invitee_email',
+        'start_date',
+        'timezone',
+        'status',
+    ];
+
+    protected $attributes = [
+        'status' => 'scheduled',
```

Comment:
Please create a class with constants somewhere in the app (ex. app/Constants/EventStatus) and put available values there.

Reply:
For what purpose i should do it?

Reply:
@dmitriy-uvin It is a good practice to use constants for such values, it will be clear which values are appropriate (values, case). The purpose is the same as for "magic numbers".